test/unit/resources/aws_cloudtrail_trail_test.rb

```diff
@@ -166,5 +172,19 @@ def describe_trails(query)
       end
       OpenStruct.new({ trail_list: [selected] })
     end
+    
+    def get_trail_status(query)
+      fixtures = [
+        OpenStruct.new({
+          name: "test-trail-1",
+          latest_cloud_watch_logs_delivery_time: TIME_NOW
```

Comment:
@dromazmj Since this is only used in one place right now, could we change `TIME_NOW` to `Time.now`?